metricbeat/module/traefik/health/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What is the purpose of this file? How is it used?

Reply:
This is an example output document that we should in the docs. It is generated by this command here: https://github.com/elastic/beats/blob/master/metricbeat/Makefile#L73 As this updates all data.json I normally run the following command:

```
go test -tags=integration github.com/elastic/beats/metricbeat/module/traefik/... -data
```
In you case this assumes the traefik docker container is running and exposes the port on localhost. This will generate a data.json file and `make update` will link it in the docs.

Here you can see an example in the docs: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-metricset-elasticsearch-index.html